src/Avalonia.Themes.Fluent/ToolTip.xaml

```diff
@@ -1,17 +1,74 @@
-<Style xmlns="https://github.com/avaloniaui" Selector="ToolTip">
-  <Setter Property="Background" Value="{DynamicResource ThemeBackgroundBrush}"/>
-  <Setter Property="BorderBrush" Value="{DynamicResource ThemeBorderMidBrush}"/>
-  <Setter Property="BorderThickness" Value="{DynamicResource ThemeBorderThickness}"/>
-  <Setter Property="Padding" Value="4,2"/>
-  <Setter Property="Template">
-    <ControlTemplate>
-      <ContentPresenter Name="PART_ContentPresenter"
-                        Background="{TemplateBinding Background}"
-                        BorderBrush="{TemplateBinding BorderBrush}"
-                        BorderThickness="{TemplateBinding BorderThickness}"
-                        ContentTemplate="{TemplateBinding ContentTemplate}"
-                        Content="{TemplateBinding Content}" 
-                        Padding="{TemplateBinding Padding}"/>
-    </ControlTemplate>
-  </Setter>
-</Style>
\ No newline at end of file
+<Styles xmlns="https://github.com/avaloniaui" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+  <Design.PreviewWith>
+      <Grid RowDefinitions="Auto,Auto"
+            ColumnDefinitions="Auto,Auto"
+            HorizontalAlignment="Center">
+        <Border Grid.Column="0"
+                Grid.Row="1"
+                Background="{DynamicResource ThemeAccentBrush}"
+                Margin="5"
+                Padding="50"
+                ToolTip.Tip="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.">
```

Comment:
I noticed that text wrapping doesn't work in ToolTip. I think it should be fixed in style or globally with separated PR.

By the way, I have tried next selector "ToolTip /template/ ContentPresenter#PART_ContentPresenter /template/ > TextBlock", but it doesn't work for me. I will take a look later on it.

Reply:
TextWrapping only works when the Tooltip is measured with a finite size

Reply:
@Gillibald Tooltip has MaxWidth=320, shouldn't it be enough for TextWrapping to get work?
I think I just couldn't set TextWrapping property from ToolTips style.

Reply:
This is the default template that is being used by the ContentPresenter: https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls/Templates/FuncDataTemplate.cs

You might want to try: `Selector="ToolTip /template/ ContentPresenter#PART_ContentPresenter > TextBlock"`